agent/internal/util/json_test.go

```diff
@@ -18,8 +18,8 @@ func TestRemoveJSONComment(t *testing.T) {
 			  "i":"o",//
 "kek": "// //"//
 }`
-	expectstr := `{				            	    	  			  "json" : true, 			  "i":"o","kek": "// //"}`
+	expectstr := `{			  "json" : true,			  "i":"o","kek": "// //"}`
```

Comment:
Test case does not cover "keeping new lines".

Reply:
True, the test case I introduced on `deply_test.go` does though.